### PR TITLE
ENCD-6105-disable-viz-btn-mouse-dev-matrix

### PR DESCRIPTION
**Pull request by zoldello**

### CODE REVIEW

**Review by forresttanaka — DISMISSED** [review]

File: src/encoded/static/components/matrix_mouse_development.js [hunk 1]
```diff
@@ -880,7 +880,7 @@ class MatrixPresentation extends React.Component {
         return (
             <>
                 <div className="matrix-header">
-                    <SearchControls context={context} visualizeDisabledTitle={visualizeDisabledTitle} additionalFilters={additionalFilters} />
+                    <SearchControls context={context} additionalFilters={additionalFilters} hideBrowserSelector />
```
Comment (zoldello) [inline src/encoded/static/components/matrix_mouse_development.js:881]: I hide the wrong button. I fixed it.

**Comment by forresttanaka** [comment]
ESLint error for an undefined variable.

File: src/encoded/static/components/matrix_mouse_development.js (hunk 1 above)
Comment (forresttanaka) [reply src/encoded/static/components/matrix_mouse_development.js:881]: Oh wow; I didn’t notice that either!

**Review by forresttanaka — APPROVED** [review]
Since I’m out tomorrow and Friday, I’ll approve this as I don’t see why it wouldn’t accomplish the goal.

**Pull request merged**